lib/api/web-element/commands/isVisible.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Returns the visible text for the element.
+ *
+ * For more info on working with DOM elements in Nightwatch, refer to the <a href="https://nightwatchjs.org/guide/writing-tests/finding-interacting-with-dom-elements.html">Finding & interacting with DOM Elements</a> guide page.
+ *
+ * @example
+ * export default {
+ *   demoTest(browser: NightwatchAPI): void {
+ *     const result = browser.element('#main ul li a.first').isVisible();
+ *       .assert.valueEquals('custom text');
+ *   },
+ *
+ *   async demoTestAsync(browser: NightwatchAPI): Promise<void> {
+ *     const result = await browser.element('#main ul li a.first').isVisible();
+ *     console.log('element text:', result);
+ *   }
+ * }
```

Comment:
If you're adding some example to JSDoc or anywhere else, it should be tested by you first by running them in one of the example test files. This example code won't work.

Reply:
alrighty ill take care of this in future